### DISCUSSION TITLE
See Assigned VLANs on the Switch Device

### DISCUSSION THREAD

**Discussion by mathias4github** [netbox-community/netbox#6153] [Q&A]
HI guys,
great tool ;-)

I'm currently documenting all our stuff. Currently I'm assigning the needed VLANs on Server End Interface. How can I see the VLAN assignments on the other Side? When I'm opening the switch side, I can't see the assignments (only add new ones)

Help is really appreciated.

Kind Regards

Mathias

**Comment by dejantep** [comment]
Are you expecting VLANs on server end to show up on switch side? That wont happen
You need to assign VLANs to switch ports aswell

**Reply by mathias4github** [reply]
ok, thank you. What is you best practise to avoid fail configuration?
